swig/python/setup.py.in

```diff
@@ -35,6 +35,9 @@ setup(
   'Topic :: Scientific/Engineering',
   ],
   package_data={
-  '@PYTHON_MODULE_NAME@':['$<TARGET_FILE_NAME:@PYTHON_MODULE_NAME@>', @FLAMEGPU_INCLUDE_CLEAN@@FLAMEGPU_PYTHON_PACKAGE_DATA_OS_SPECIFIC@],
+  '@PYTHON_MODULE_NAME@':['$<TARGET_FILE_NAME:@PYTHON_MODULE_NAME@>', @FLAMEGPU_CODEGEN_INCLUDE_CLEAN@@FLAMEGPU_INCLUDE_CLEAN@@FLAMEGPU_PYTHON_PACKAGE_DATA_OS_SPECIFIC@],
   },
+  install_requires=[
+    'six', 'astpretty',
```

Comment:
`six` should not be a required dependency as we do not care for python 2 compatibility. See previous comments.

Reply:
Gone